src/main/resources/hudson/plugins/warnings/WarningsPublisher/global.jelly

```diff
@@ -1,18 +1,18 @@
-<?jelly escape-by-default='true'?>

```

Comment:
All the lines in this file had MSDOS EOL codes (\r\n) at their ends.
...except the last line, which had no EOL code.